components/header.tsx

```diff
@@ -12,8 +12,10 @@ const Header = ({ onMenuClick }: { onMenuClick: () => void }) => {
   return (
     <div className="fixed w-full flex justify-between items-center py-4 px-4 sm:px-8 bg-white bg-opacity-5 backdrop-blur">
       <div className="flex items-center gap-12">
-        <LogoSvg className="h-9 hidden sm:flex w-fit" />
-        <LogotypeSvg className="h-4 sm:hidden w-fit" />
+        <a target="_blank" href="/" rel="noreferrer">
```

Comment:
We should use the `<Link>` component for next.js

Reply:
https://nextjs.org/docs/pages/api-reference/components/link

Reply:
https://github.com/clober-dex/coupon.finance/pull/36/commits/f6ba5c0757dd5bf0edf1cc2a14caa0d960cd571a